tests/unit/components/ObsDetails/ObsDetails.test.js

```diff
@@ -63,17 +64,27 @@ jest.mock( "../../../../src/components/LoginSignUp/AuthenticationService", ( ) =
 
 jest.mock( "components/ObsDetails/AddCommentModal" );
 
+jest.mock( "sharedHooks/useIsOnline" );
```

Comment:
I think you need to use fakeTimers for this test then, because netInfo uses timers when checking internet connection. But you would need to check if that is true.